src/components/Settings.js

```diff
@@ -16,1298 +33,1707 @@ const bibUtil_to_json = require(`${__dirname}/../util/usfm_to_json`);
 const path = require("path");
 const Promise = require("bluebird");
 var fs = Promise.promisifyAll(require("fs"));
+var appPath = path.join(__dirname, "..", "..");
+let flag = false;
 
 @observer
 class SettingsModal extends React.Component {
-    constructor(props) {
-        super(props);
+  constructor(props) {
+    super(props);
 
-        this.state = {
```

Comment:
Both @Ashersam and @vipinpaul, please follow the same conventions in your editor for spacing. Such commits with mostly formatting changes are distracting.

Reply:
Did auto code formatting so the spacing has changed, will revert the code and update it, and push it soon.